badgerTrie/examples/accusation_builder_test.go

```diff
@@ -781,13 +839,46 @@ func TestGenerateAccusations(t *testing.T) {
 		t.Fatal(err)
 	}
 	preImageBin, err := tx2.Vin[0].TXInLinker.TXInPreImage.MarshalBinary()
+
+	vsImageBin, err := tx2.Vout[0].MarshalBinary()
+	txOutIdx_, err := tx2.Vout[0].TxOutIdx()
+	chainId_, err := tx2.Vout[0].ChainID()
+	value_, err := tx2.Vout[0].Value()
+	h := fmt.Sprintf("%x", value_)
+	genericOwner_, err := tx2.Vout[0].GenericOwner()
+	utxoId_, err := tx2.Vout[0].UTXOID()
+	log.Printf("\n\n ======== VS Preimage bin: ======= \n%x\n", vsImageBin)
```

Comment:
Should use `t.Logf`, which will only print log output if `go test` is passed the `-v` flag, or if the test fails. Keeps console output cleaner.

Reply:
Changed as suggested